camayoc/tests/qcs/api/v1/scans/test_network_scans.py

```diff
@@ -136,10 +87,11 @@ def prep_network_scan(source, cleanup, client):
     return scan
 
 
-@pytest.mark.parametrize('source', sources(), ids=[
+@pytest.mark.create
```

Comment:
I am curious about this create mark? What is the plan?

Reply:
This was a convenience for me to run just one of the scan tests for each type. I can remove it.

Reply:
You can do that using the `-k` by running `py.test  -k "create" camayoc/tests/api/v1/`. I mean it is fine to have those but I think we should look for consistency and maybe consider adding to other tests if we decide that is the way to go